controls/anssi.yml

```diff
@@ -0,0 +1,442 @@
+policy: 'ANSSI-BP-028'
```

Comment:
Nice catch! I confirm using BP is correct for the latest version. The profile files should be renamed, then.